src/components/Actions/Actions.vue

```diff
@@ -497,6 +563,13 @@ export default {
 			type: Boolean,
 			default: false,
 		},
+		/**
+		 * Enable popover focus trap
+		 */
+		focusTrap: {
```

Comment:
Do we need this prop? If it works, is there a reason to disable the focus-trap?

Reply:
yes, to make sure we can handle any edge case